environs/bootstrap/bootstrap.go

```diff
@@ -199,13 +206,15 @@ func bootstrapCAAS(
 	if err != nil {
 		return errors.Trace(err)
 	}
-	// TODO(caas): how to find the best/newest jujud docker image to use
-	newestTool := version.MustParseBinary("2.5-beta1-bionic-amd64")
+
+	// TODO(caas): how to find the best/newest jujud docker image to use.
```

Comment:
Remove this TODO as we always match the docker image to juju version

Reply:
removed